nevergrad/optimization/optimizerlib.py

```diff
@@ -977,6 +1015,21 @@ def _internal_tell_not_asked(self, candidate: p.Parameter, loss: tp.FloatLoss) -
             opt.tell(local_candidate, loss)
 
 
+class ConfRescaleOptimizer(base.ConfiguredOptimizer):
+    """
```

Comment:
please add the docstring here.
I would probably just name this `Rescaled` and hide the base optimizer

Reply:
Done.